proto/tendermint/abci/types.proto

```diff
@@ -435,7 +435,7 @@ message Validator {
 }
 
 message ValidatorUpdate {
-  tendermint.crypto.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
+bytes pub_key = 1 ;
```

Comment:
```suggestion
  bytes pub_key = 1;
```

formatting seems off

Reply:
Also breaks the type without changing the field number.